metaci/release/models.py

```diff
@@ -51,6 +53,7 @@ def get_default_production_date():
 
 class DefaultImplementationStep(BaseModel):
     start_date_offset: int = 0
+    push_time: int
```

Comment:
This should have a default value of None so that we aren't forced to enter it for default implementation steps that aren't push upgrades.